<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_setConfig</class>
 <widget class="QDialog" name="Dialog_setConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>配置</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器地址：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_serverip">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_setconfig">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>配置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
